src/components/CompareResults/CommonGraph.tsx

```diff
@@ -293,8 +293,8 @@ function CommonGraph({ baseValues, newValues, unit }: CommonGraphProps) {
       extent: [min, max],
     }),
   );
-  //////////////////// END FAST KDE   ////////////////////////
 
+  //////////////////// END FAST KDE   ////////////////////////
```

Comment:
Is there a reason why this comment was moved?

Reply:
Oh i think it just moved a line lower.

Reply:
ill remove the line space